dns/forms.py

```diff
@@ -58,3 +58,4 @@ def _rev_to_ip(self, name):
 
     class Meta:
         model = Record
+        exclude = []
```

Comment:
There seems to be a security warning against this approach:

https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#selecting-the-fields-to-use